components/payments/internal/storage/balances_test.go

```diff
@@ -0,0 +1,480 @@
+package storage
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/formancehq/go-libs/bun/bunpaginate"
+	"github.com/formancehq/go-libs/logging"
+	"github.com/formancehq/go-libs/pointer"
+	"github.com/formancehq/go-libs/time"
+	"github.com/formancehq/payments/internal/models"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	defaultBalances = []models.Balance{
```

Comment:
If you stored these in a map (`map[accID][]models.Balance`) you could reference them directly later in your tests rather than having to redefine `expectedBalances` in every test.

It might also make it more obvious why you expect certain datasets in your query balance requests.